examples/game_of_life/Cargo.toml

```
@@ -7,7 +7,7 @@ authors = ["Diego Cardoso <dige0card0s0@hotmail.com>",
 edition = "2018"
 
 [dependencies]
-rand = "0.7.3"
 log = "0.4"
 wasm-logger = "0.2.0"
 yew = { path = "../../yew" }
+getrandom =  { version = "*", features = ["wasm-bindgen"] }
```

It's better to use a fixed version of this because otherwise it will cause problems if an update to `getrandom` is released.
```suggestion
getrandom =  { version = "0.1", features = ["wasm-bindgen"] }
```